### ISSUE TITLE
Request: configure default axes positions and autoscale

### ISSUE THREAD

**Issue by acorderob** [mcmonkeyprojects/sd-infinity-grid-generator-script#52]
Add settings in the yaml file to set which axes are by default x1, y1, x2 (super), y2 (super), and which value should the auto-scale option be.

**Comment by mcmonkey4eva** [comment]
Added, under `grid`:
```yml
    # Optionally, also specify default choices for the webpage.
    show descriptions: true
    autoscale: false
    sticky: false
    x axis: sampler
    y axis: steps
    x super axis: 'None'
    y super axis: 'None'
```

**Comment by mcmonkey4eva** [comment]
it is in the readme, under <https://github.com/mcmonkeyprojects/sd-infinity-grid-generator-script#1-grid-definition-file>
```
It can optionally define show descriptions, autoscale, and sticky as true or false to change default web-viewer settings.
It can optionally define x axis, y axis, x super axis, and y super axis as axis IDs to change the default web-viewer axes.
```

**Comment by Feanix-Fyre** [comment]
I haven't yet been able to get it to work but I'm going to assume I'm doing something wrong, as opposed to something being wrong with the advice. I've just been editing the data.js output file to make the html look the way I want it each time cause that was easiest for me.

**Comment by mcmonkey4eva** [comment]
If you can't get it working, post a `yml` file you tried?

**Comment by Feanix-Fyre** [comment]
It's been a while so it'll be difficult to find one. I had tried both the megagrid.yml format and the short_example.yml format. I followed the formats exactly. I copy pasted what you had suggested below directly under grid and changed the values of x axis/y axis/x super axis/ y super axis to, for example, Sampler or sampler (or another category as shown in megagrid.yml. I tried both upper-case and lower-case. It wouldn't adjust the default axes when I loaded the webpage.

I'm providing this information in case it affects anybody else. I have made alterations to my procedure and methods in order to achieve my goals, so I don't need your attention on the matter. That being said, I am grateful for your attention and responses.

> Added, under `grid`:
> 
> ```yaml
>     # Optionally, also specify default choices for the webpage.
>     show descriptions: true
>     autoscale: false
>     sticky: false
>     x axis: sampler
>     y axis: steps
>     x super axis: 'None'
>     y super axis: 'None'
> ```
